src/sqLiteBase64.ts

```diff
@@ -0,0 +1,2 @@
+export const SQLBASE64 =
```

Comment:
I think this belongs in `vite-plugin-inline` that deals with encoding and compression. Has the additional advantage that `base85` instead of `base64` is used and that the content is gzipped. Should make reports smaller. Once we migrate from `gzip` to `zstd` in `vite-plugin-inline` this will be included as well.